package.json

```diff
@@ -10,8 +10,8 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
-    "lint": "eslint --ext .js,.jsx ./",
+    "build": "react-scripts build && cp build/index.html build/404.html",
+    "lint": "eslint ./src --ext .js,.jsx",
```

Comment:
only lint the `/src` directory – I realized once I ran a build that it tried to lint all the output minified JS files 😄